doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -555,6 +551,21 @@ Bluetooth samples
 
   * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
 
+Bluetooth Fast Pair samples
+---------------------------
+
+* Defined the new :ref:`bt_fast_pair_samples` category and moved the :ref:`fast_pair_input_device` sample into it.
+
+* :ref:`fast_pair_input_device` sample:
+
+  * Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
+  * Replaced the :ref:`zephyr:bluetooth-hci-ipc-sample` radio core firmware with the :ref:`ipc_radio`.
+  * Removed unnecessary :kconfig:option:`CONFIG_HEAP_MEM_POOL_SIZE` Kconfig configuration.
+
+* Added :ref:`fast_pair_locator_tag` sample to demonstrate support for the locator tag use case and
+the Find My Device Network (FMDN) extension.
+  The new sample is available in the debug build configuration (:file:`prj.conf`) and the release build configuration (:file:`prj_release.conf`).
```

Comment:
Consider: I wonder if it would not be better to refer to the configuration sets using the eventual file suffixes instead of the main application configuration file.

Reply:
Hmm, both variants highlight different things. The current one is more user-friendly in terms of navigation to the file to see the release configuration. Your suggestion is more user-friendly in terms of testing the release configuration in runtime.

Let's see input from others.

BTW, could you request a specific change so that we can evaluate the counter proposal? Would it look somewhat similar to this?

```suggestion
  The new sample is available in the debug build configuration (the default option) and the release build configuration (`-DFILE_SUFFIX=release`).
```

Reply:
Well, release configuration can consist of multiple files in addition to the main application configuration file, thats why i suggested it to refer to file suffixes (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/develop/application/index.html#application-file-suffixes) based on the experience in the nRF Desktop app.
However, this sample is not that complicated so I will not insist on that, just suggestion that shows which files to search and how you can build that.

I think your proposal look fine, or simply just
```suggestion
  The new sample is available in the debug build configuration (the default option) and the release build configuration (the ``release`` file suffix).
```

Reply:
fixed with small changes